96-11-using-config-r.md

```diff
@@ -35,6 +35,16 @@ In some cases, authors provide us privately with data that is not part of the pu
 s-drive <- "L:/Workspace/aearep-9999-implicit-nda"
 ```
 
+:::{note}
+
+If you are working on Windows (e.g. CCSS-Cloud) then you would need to use `\` or `//` to write filepaths or use the file.path() function. So, for example, the above would become:
+
+```
+s-drive <- "L://Workspace//aearep-9999-implicit-nda"
```

Comment:
This is not true. Simple forward slashes work. And specifically, it cannot be `\` because that's what failed to work in your case.

Reply:
Sorry I meant double backslash since that worked for me after going on stack exchange to run the code prior.